src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -67,18 +68,36 @@ public override void Startup(ViewStartupParams viewStartupParams)
             {
                 var docsDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, FALLBACK_DOC_DIRECTORY_NAME));
                 fallbackDocPath = docsDir.Exists ? docsDir : null;
+
+                //When executing DynamoSandbox the WebView2 cache folder will be located in the same directory than the executable
+                var userDataDir = new DirectoryInfo(pathManager.DynamoCoreDirectory);
+                webBrowserUserDataFolder = userDataDir.Exists ? userDataDir : null;
             }
 
             if (!string.IsNullOrEmpty(pathManager.HostApplicationDirectory))
             {
-                var docsDir = new DirectoryInfo(Path.Combine(pathManager.HostApplicationDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                //when running over Revit HostApplicationDirectory = C:\Program Files\Autodesk\Revit 2023\AddIns\DynamoForRevit\Revit
+                //Then we need to remove the Revit folder from the path so we can find the fallback_docs directory.
+                var hostAppDirectory = Directory.GetParent(pathManager.HostApplicationDirectory).FullName;
```

Comment:
We may want to double check on this if there is a better way. I will get back to you

Reply:
So for Sandbox, this pathManager.HostApplicationDirectory would be null? I think this only means this is not an in process integration, for case like FormIt, where it is out of process but still delivered in programe files, this may still not work

Reply:
@QilongTang 
For other products like FormIt or Civil not sure which is the path of the Dynamo "fallback_doc" folder, I followed the implementation that I found in PackageDocumentationManager.cs (check link below) then if the fallback_doc folder location is different for each host (Revit, Civil, Formit) then the implementation done in PackageDocumentationManager is not working or hostDynamoFallbackDocPath = null  or there is another process creating the folder and moving all the md/jpg files to the HostApplicationDirectory path.

Should I install FormIt and Civil so I can test if images are shown in DocumentationBrowser?

https://github.com/DynamoDS/Dynamo/blob/79ebac3289a4474b4d3cba4d2280ed349ce6c710/src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs#L66

Reply:
@QilongTang now the WebView2 cache folder will be always created in AppData, see the next commit:
https://github.com/DynamoDS/Dynamo/pull/13245/commits/e101e8a2699fc306239ef702b8ae408ba54a3078

Reply:
@QilongTang fixed comment about Revit in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13245/commits/2729c08968deabdad13b23ee962bae040d8c2fd1